systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingST.java

```diff
@@ -553,17 +608,65 @@ void testProducerConsumerMirrorMakerConnectStreamsService() {
                 .endSpec()
                 .done();
 
+        TracingUtils.verify(JAEGER_PRODUCER_SERVICE, kafkaClientsPodName, "To_" + TOPIC_NAME);
+        TracingUtils.verify(JAEGER_CONSUMER_SERVICE, kafkaClientsPodName, "From_" + TOPIC_NAME);
+        TracingUtils.verify(JAEGER_MIRROR_MAKER_SERVICE, kafkaClientsPodName, "From_" + TOPIC_NAME);
+        TracingUtils.verify(JAEGER_MIRROR_MAKER_SERVICE, kafkaClientsPodName, "To_" + TOPIC_NAME);
+
+        LOGGER.info("Deleting topic {} from CR", TOPIC_NAME);
+        cmdKubeClient().deleteByName("kafkatopic", TOPIC_NAME);
+        KafkaTopicUtils.waitForKafkaTopicDeletion(TOPIC_NAME);
+
+        LOGGER.info("Deleting topic {} from CR", TOPIC_NAME + "-target");
+        cmdKubeClient().deleteByName("kafkatopic", TOPIC_NAME + "-target");
+        KafkaTopicUtils.waitForKafkaTopicDeletion(TOPIC_NAME + "-target");
+    }
+
+    @Test
+    @Tag(CONNECT)
+    @Tag(MIRROR_MAKER)
+    @Tag(CONNECT_COMPONENTS)
+    @SuppressWarnings({"checkstyle:MethodLength"})
+    void testProducerConsumerMirrorMakerConnectStreamsService() {
+        Map<String, Object> configOfKafka = new HashMap<>();
+        configOfKafka.put("offsets.topic.replication.factor", "1");
+        configOfKafka.put("transaction.state.log.replication.factor", "1");
+        configOfKafka.put("transaction.state.log.min.isr", "1");
```

Comment:
Is this specific configuration really needed?

Reply:
The GitHub diff might be confusing, but this is your code :-D. I can remove it and see if it still works.